src/platform/imx8m/include/platform/drivers/interrupt.h

```diff
@@ -77,6 +76,35 @@
  */
 int irqstr_get_sof_int(int irqstr_int);
 
+#define IRQSTR_BASE_ADDR	0x30A80000
+
+/* The MASK, SET (unused) and STATUS registers are 160-bit registers
+ * split into 5 32-bit registers that we can directly access.
+ *
+ * The interrupts are mapped in the registers in the following way:
+ * Interrupts 0-31 at offset 0
+ * Interrupts 32-63 at offset 1
+ * Interrupts 64-95 at offset 2
+ * Interrupts 96-127 at offset 3
+ * Interrupts 128-159 at offset 4
+ */
+
+/* This register is only there for HW compatibility; on this platform it
+ * does nothing.
+ */
+#define IRQSTR_CHANCTL			0x00
+
+#define IRQSTR_CH_MASK(n)		(0x04 + 0x04 * (n))
+#define IRQSTR_CH_SET(n)		(0x18 + 0x04 * (n))
+#define IRQSTR_CH_STATUS(n)		(0x2C + 0x04 * (n))
+#define IRQSTR_MASTER_DISABLE		0x40
+#define IRQSTR_MASTER_STATUS		0x44
+
+#define IRQSTR_RESERVED_IRQS_NUM	0
+#define IRQSTR_IRQS_NUM			160
```

Comment:
IRQSTR_IRQS_NUM indentation looks strange. Or is it the github's fault?

Reply:
Same explanation as in the other case, the "+ " from diff format makes tabs no longer line up.